src/renderer/components/data-settings/data-settings.js

```diff
@@ -228,6 +234,75 @@ export default Vue.extend({
       this.handleFreetubeImportFile(filePath)
     },
 
+    handleYoutubeCsvImportFile: function(filePath) { // first row = header, last row = empty
```

Comment:
I wonder if this comment should be placed just before `for (let i = 1; i < (youtubeSubscriptions.length - 1); i++)` 
As it would explain the starting value being `1` not `0`

Reply:
I agree, that comment should be moved to be before the for loop.